src/dashboard/portal/dashboard_portal_ui_farming.md

```diff
@@ -25,6 +25,12 @@ If you have public IPv4 addresses available that can be used for usage on the TF
 Click `ADD IP`, specify the addresses, the gateway and click `CREATE`.
 You can add them one by one or using range of IPs.
 
+__Some notes about adding a new IPs__:
+
+- Make sure to not add a new IP range that includes an already existing ip address, this will cause an error.
```

Comment:
- Be careful not to create a new IP range that contains an IP address that already exists; doing so will result in an error.
- Verify that both the gateway address and the IP address are correct.
- Be careful not to include the same gateway address in a new IP range.